File: Controls/Controls/KeyboardPopup.vb
Imports System.Windows.Input

Public Class KeyboardPopup
  Inherits Windows.Controls.Primitives.Popup

  Protected Overrides Sub OnOpened(e As EventArgs)
    MyBase.OnOpened(e)

    'MOVE FOCUS INTO POPUP WHEN OPENED
    Me.Child.MoveFocus(New TraversalRequest(FocusNavigationDirection.Next))
  End Sub

End Class

File: CSharpControls/Controls/KeyboardPopup.cs
using System;      
using System.Windows.Input;

namespace CSharpControls
{
  public class KeyboardPopup : System.Windows.Controls.Primitives.Popup
  {
    protected override void OnOpened(EventArgs e)
    {
      base.OnOpened(e);
      this.Child.MoveFocus(new TraversalRequest(FocusNavigationDirection.Next));
    }
  }
}